js/faq.test.js

```diff
@@ -0,0 +1,77 @@
+const removeFocusForOthers = require('./faq');
+
+describe('handle keydown event', () => {
+  let ancTag, event;
+  beforeEach(() => {
+    ancTag = document.createElement('a');
+    ancTag.setAttribute('href', '/example');
+    ancTag.click = jest.fn();
+    document.body.appendChild(ancTag);
+    event = new KeyboardEvent('keydown', { key: ' ' });
+  });
+
+  it('should not call click on anchor tag when keydown event is not space', () => {
+    event.key = 'Enter';
+    event.target = ancTag;
+    window.dispatchEvent(event);
+    expect(ancTag.click).not.toHaveBeenCalled();
+  });
+  it('should not call click on anchor tag when anchor tag is not found', () => {
+    window.dispatchEvent(event);
+    expect(ancTag.click).not.toHaveBeenCalled();
+  });
+});
+
+test('removes focus for all elements except the target', () => {
```

Comment:
instead of `test` we can call `it` as used in other places

Reply:
yes I have done